tests/raftstore/cluster.rs

```diff
@@ -146,19 +146,42 @@ impl<T: Simulator> Cluster<T> {
         self.sim.rl().call_command(request, timeout)
     }
 
+    fn naive_call_command_on_leader(&mut self,
```

Comment:
naive confuses me, why naive? 

I think you can move this to call_command_on_leader directly.